src/test/js/shared/random_data.js

```diff
@@ -44,6 +53,7 @@ module.exports = {
     getRandomEmailAddress: () => randomString() + "@mailtest.gov.uk",
     getRandomUserPassword: () => generatePassword(12),
     getRandomClientSecret: () => generatePassword(12),
+    getRandomTextFor11KB: () => createRandomString(1112150),
```

Comment:
I think 1112150 bytes would be ~1 MB :) 

(Also - a char is 2 bytes so we end up creating a 2MB cookie:) )

Reply:
Now its been set around 11 KB